gcp/appengine/frontend3/src/templates/list.html

```diff
@@ -14,9 +14,11 @@
     <div class="mdc-data-table__header-cell-label">
       {{ column_name }}
     </div>
-    {% if is_sortable %}
-    <mwc-icon-button class="mdc-data-table__sort-icon-button" icon="arrow_upward" disabled
-      aria-label="Sort by {{ column_name }}" aria-describedby="{{ column_id }}-status-label"></mwc-icon-button>
+    {% if is_sorted %}
+    <md-icon-button class="mdc-data-table__sort-icon-button" disabled
+      aria-label="Sorted by {{ column_name }}" aria-describedby="{{ column_id }}-status-label">
+      <md-icon aria-hidden="false">arrow_upward</md-icon>
+    </md-icon-button>
```

Comment:
As far as there is not sortable column, the if statement should check the `is_sorted` rather that `is_sortable`. 
This change also fixed the color of the arrow to be grey instead of black to give more visibility.
Before:
![Screenshot 2024-06-12 at 3 30 44 PM](https://github.com/google/osv.dev/assets/73332835/2f032947-7456-4dc1-8cd1-5550e893ee9b)


After:
<img width="160" alt="Screenshot 2024-06-14 at 12 51 27 PM" src="https://github.com/google/osv.dev/assets/73332835/40497b2d-5864-49fd-b47f-e4ef45f4d2f9">